filters.go

```diff
@@ -72,10 +72,13 @@ func WhiteListHostFilter(hosts ...string) HostFilter {
 
 	m := make(map[string]bool, len(hostInfos))
 	for _, host := range hostInfos {
-		m[host.ConnectAddress().String()] = true
+		connAddr, _ := host.ConnectAddress()
```

Comment:
if `err != nil` I don't think it should be whitelisted, it should just `continue`